amundsen_application/static/js/components/DashboardPage/ImagePreview/index.tsx

```diff
@@ -34,36 +61,58 @@ export class ImagePreview extends React.Component<ImagePreviewProps, ImagePrevie
     this.setState({ isLoading: false, hasError: true });
   }
 
-  render = () =>  {
+  handlePreviewButton = () => {
+    this.setState({ isModalVisible: true });
+  }
+
+  handlePreviewModalClose = () => {
+    this.setState({ isModalVisible: false });
+  }
+
+  render = () => {
     const { uri, redirectUrl } = this.props;
+    const { isLoading, hasError, isModalVisible } = this.state;
+    const imageSrc = `${Constants.PREVIEW_BASE}/${uri}/${Constants.PREVIEW_END}`;
+
     return (
-      <div className='image-preview'>
-        {
-          this.state.isLoading &&
+      <div className="image-preview">
+        {isLoading && (
           <div className="text-placeholder">
```

Comment:
Slipped my mind in your shimmer loader PR, but we no longer need the `<div>` to apply the `text-placeholder` style on this, since the loading state no longer has text.

Reply:
Makes sense, removing!